classes/classes/Creature.as

```diff
@@ -403,7 +403,7 @@ package classes
 		//Fertility is a % out of 100. 
 		public var fertility:Number = 10;
 		public var nippleLength:Number = .25;
-		public var breastRows:Array;
+		public var breastRows:/*BreastRowClass*/Array;
```

Comment:
I'm surprised this even compiles...

A good reason to try and replace `Array` with `Vector` where possible (for readability and type safety).